tests/drivers/uart/uart_async_api/testcase.yaml

```diff
@@ -61,6 +61,10 @@ tests:
     filter: CONFIG_SERIAL_SUPPORT_ASYNC and CONFIG_UART_MCUX_LPUART
     harness: ztest
     depends_on: dma
+    extra_configs:
```

Comment:
Will this work? Some platforms (like the K66 or K8X) do not have support for NOCACHE memory (no ARMv7 MPU), but do use the LPUART